### PR TITLE
Add instructions on running mkdocs locally

### PR DESCRIPTION
**Pull request by simkimsia**
for the ignorant. Ref https://github.com/igordejanovic/textX/issues/143#issuecomment-449556703

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -41,6 +41,25 @@ textX could always use more documentation, whether as part of the
...
+3. Create a virtualenv for the fork and install the relevant libraries (one-time effort)
+
+Once you complete the above 3 instructions, you can now:
+
+4. Activate the virtualenv
+5. Run `mkdocs serve` at the root folder
+
+`mkdocs` will run a webserver that serves the documentation at 127.0.0.1:8000
+
+To make changes to the configurations, you can look at `mkdocs.yml`. For more information on how to use mkdocs, visit this [site](https://www.mkdocs.org). Essentially, you do not need to install mkdocs as a software locally to test the documentations.
```
Comment (igordejanovic) [inline CONTRIBUTING.md]: Not sure what do you mean by `you do not need to install mkdocs as a software locally...`. You actually do need mkdocs. It is installed as a part of `pip install -r requirements_dev.txt`. See the file, there is `mkdocs` in it. I suggest to remove this sentence entirely as it is confusing.

**Review by igordejanovic — CHANGES_REQUESTED** [review]
Looks good. Just one change is required at the end. Please, see below.

File: CONTRIBUTING.md (hunk 1 above)
Comment (simkimsia) [reply CONTRIBUTING.md]: Understood. Will make another change and then squash the commits. I have to go get dinner. Then I will come back and fix this asap.

**Review by goto40 — COMMENTED** [review]
Note : when serving locally some files get generated. Can you check .gitignore if the local docu is correctly ignored.

OK. Just checked. Seems to be OK.

File: CONTRIBUTING.md (hunk 1 above)
Comment (igordejanovic) [reply CONTRIBUTING.md]: Sure. No rush :)

Comment (simkimsia) [reply CONTRIBUTING.md]: @igordejanovic done. I force pushed the squash. It should be updated soon.

**Comment by igordejanovic** [comment]
Thanks. Merging.

**Pull request merged**